templates/AureliaSpa/ClientApp/app/components/app/app.html

```diff
@@ -0,0 +1,14 @@
+<template>
+    <require from="../navmenu/navmenu.html"></require>
+    <require from="./app.css"></require>
+    <div class='container-fluid'>
+        <div class='row'>
+            <div class='col-sm-3'>
+                <navmenu router.bind="router"></navmenu>
+            </div>
+            <div class='col-sm-9 body-content'>
+                <router-view></router-view>
+            </div>
+        </div>
+    </div>
+</template>
```

Comment:
Missing trailing linebreak which is present in most files. Could you add one here and to any other TS/JS/HTML/CSS file where it's missing?